packages/garbo/src/tasks/post/index.ts

```diff
@@ -294,6 +294,16 @@ function leafResin(): GarboPostTask {
   };
 }
 
+function wardrobeOMatic(): GarboPostTask {
+  return {
+    name: "Wardrobe-o-Matic",
+    available: have($item`wardrobe-o-matic`),
+    ready: () => myLevel() >= 20,
```

Comment:
maybe lower the threshold if ascending? after i cs loop i usually enter day 2 at like lv17 and i don't think usually even hit 20 by eod.